pydantic_ai_slim/pydantic_ai/ext/aci_tools.py

```diff
@@ -0,0 +1,82 @@
+from aci import ACI
+
+def clean_schema(schema):
+    if isinstance(schema, dict):
+        # Remove non-standard keys (e.g., 'visible')
+        return {k: clean_schema(v) for k, v in schema.items() if k not in {'visible'}}
+    elif isinstance(schema, list):
+        return [clean_schema(item) for item in schema]
+    else:
+        return schema
+
+def tool_from_aci(aci_function: str,
+                  linked_account_owner_id: str
+                  ) -> Tool:
```

Comment:
Can you have a look at the linter errors? The indentation is a bit off here. You'll want to run `make install` and then `make format` to auto-format everything.